src/core/turn-order.js

```diff
@@ -44,8 +44,34 @@ export const TurnOrder = {
    * The default round-robin turn order.
    */
   DEFAULT: {
-    first: (G, ctx) => ctx.currentPlayer,
-    next: (G, ctx) => (+ctx.currentPlayer + 1) % ctx.numPlayers + '',
+    order: (G, ctx) => {
```

Comment:
Let's not change `DEFAULT`, or any of the other turn orders in this file. How about creating a new one:

```
CUSTOM: {
  first: G => G.playOrder[0] + '',
  next: (G, ctx) => ...
}
```

I don't understand the purpose of `order`.

Also, we should probably make the fieldname `playOrder` customizable. Maybe make `CUSTOM` accept a parameter as the field name and have it return the turn order implementation?

```
CUSTOM: (fieldname = 'turnOrder') => ({
  first: G => G[fieldname][0] + '',
  next: (G, ctx) => ...
})
```

which can be used like:

```
Game({
  moves: { ... }
  flow: {
    turnOrder: TurnOrder.CUSTOM(),
  }
})
```